JAICore/jaicore-search/src/main/java/jaicore/search/problemtransformers/GraphSearchProblemInputToGraphSearchWithSubpathEvaluationInputTransformerViaRDFS.java

```diff
@@ -31,24 +33,30 @@ public GraphSearchProblemInputToGraphSearchWithSubpathEvaluationInputTransformer
 	}
 
 	public INodeEvaluator<N, V> getPreferredNodeEvaluator() {
-		return preferredNodeEvaluator;
+		return this.preferredNodeEvaluator;
 	}
 
 	public Predicate<N> getPrioritizedNodePredicatesForRandomCompletion() {
-		return prioritizedNodesInRandomCompletion;
+		return this.prioritizedNodesInRandomCompletion;
 	}
 
 	public int getSeed() {
-		return seed;
+		return this.seed;
 	}
 
 	public int getNumSamples() {
-		return numSamples;
+		return this.numSamples;
 	}
 
 	@Override
-	public GraphSearchWithSubpathEvaluationsInput<N, A, V> transform(GraphSearchWithPathEvaluationsInput<N, A, V> problem) {

```

Comment:
In this method, the preferred node evaluator is completely ignored, when the transform method is called.